KsApi/models/GraphCategory.swift

```diff
@@ -0,0 +1,132 @@
+import Foundation
+
+public struct ParentCategory: Swift.Decodable {
+  public let id: String
+  public let name: String
+
+  public var categoryType: RootCategoriesEnvelope.Category {
+      return RootCategoriesEnvelope.Category(
+        id: id,
+        name: name
+    )
+  }
+}
+
+public struct RootCategoriesEnvelope: Swift.Decodable {
+  public let rootCategories: [Category]
+
+  public struct Category: Swift.Decodable {
+    public let id: String
+    public var intID: Int? {
+        return decompose(id: id)
+    }
+
+    public static let gamesId: Int = 12
+
+    public let name: String
+    public let parentId: String?
+    internal let parentCategory: ParentCategory?
+    public var _parent: RootCategoriesEnvelope.Category? {
```

Comment:
Was this previously exposed as `_parent`? We could make the `_parent` `private` and expose `parent`, we'd just need to use `CodingKeys` to decode it to the correct property. Of course that would mean implementing the `Decodable` initializer but might be worth not having to refer to `_parent` where it's use in the app... 🤔

Reply:
I agree!